### ISSUE TITLE
Add initial configuration to Gihub repository

### ISSUE THREAD

**Issue by yuval-linker** [platanus/potassium#342]
When a user selects to create a Github repository it could be beneficial to have the option to give an initial configuration to the repo. Like needing a review on every pull request to merge.

**Comment by rjherrera** [comment]
@yuval-linker I agree, this would be a nice addition. I'm not sure if the gem we use to generate repos in GitHub allows this kind of configuration, but it would be nice to enforce the reviewers thingy alongside the automatic deletion of stale branches. We could have a "standard config" for our recommended way to handle repositories.

**Comment by agomezcampero** [comment]
I think it would be a good addition specially for Platanus. I wouldn't add the configurations as a user option, but rather enforce some basic options we currently use. In my opinion there are enough questions already while creating a new project and adding Github repo options seems a bit excessive, adding a few options that are not too specific sounds like a good idea.
